run.py

```diff
@@ -0,0 +1,5 @@
+from star_tides.create_app import create_app
+
+
+if __name__ == '__main__':
+    create_app().run(host='0.0.0.0', port=5000)
```

Comment:
Up to you: make this an env var?

Reply:
I don't think this would ever change. When we implement cloud infra, we just have to make sure the port matches.